app.go

```diff
@@ -135,6 +135,13 @@ func (s *App) Stop(ctx context.Context) error {
 	return withTimeout(ctx, s.lifecycle.stop)
 }
 
+// Done allows blocking on SIGINT or SIGTERM
+func (App) Done() <-chan os.Signal {
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, syscall.SIGINT, syscall.SIGTERM)
+	return c
+}
```

Comment:
Merge train seems to have sailed, I think we should consider a different signature in the final release. Rather than exposing `<-chan os.Signal`, we may want to follow the lead of `context.Context` and expose `<-chan struct{}`. This also gives us more flexibility down the road.